### DISCUSSION TITLE
Problems upgrading to 1.27

### DISCUSSION THREAD

**Discussion by ewrible** [teslamate-org/teslamate#2780] [General]
I thought I did everything right, added the key to the file but when I checked today I am unable to access Teslamate.  Looking at the log, I keep getting the following.  Anyone have an idea?

It's like the key isn't being accepted, not really sure what's going on or how to course correct.

https://www.icloud.com/iclouddrive/096fODvuGUUX0K-FFYXeCRmtw#tm

**Comment by adriankumpf** [comment]
Does this work for you: https://github.com/adriankumpf/teslamate/issues/2760#issuecomment-1187582499
